chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,870 @@
+    let runaway_max = 3;
+    while (text.indexOf('%') !== -1 && runaway_max-- > 0) {
+      replacify();
+    }
+
+    return text;
+  }
+
+  replacer_onEach(text, re, new_text, value, key) {
+    re = new RegExp(`%${this.escapeRegExp(key)}%`, 'gi');
+    new_text = text.replace(re, value);
+    text = new_text;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)